packages/preview/quetta/0.2.0/README.md

```diff
@@ -0,0 +1,73 @@
+A simple module to write [tengwar](https://en.wikipedia.org/wiki/Tengwar) in [Typst](https://typst.app/).
+
+## Requirements
+
+- [Typst](https://github.com/typst/typst) version 1.11.0 or 1.11.1
```

Comment:
I think you meant `0.11.0` or `0.11.1`.

Reply:
Oops! Indeed! Thanks for having noticed that!